src/components/votes/votesTab.js

```diff
@@ -154,7 +156,10 @@ class VotesTab extends React.Component {
                     : '-'}
                 </div>
                 <div className={`${grid['col-sm-2']} ${grid['col-lg-1']}`}>
-                  {vote.productivity !== undefined ? `${vote.productivity}%` : '-'}
+                  {vote.productivity !== undefined
+                    ? `${formatAmountBasedOnLocale({ value: vote.productivity, locale: i18n.language })}%`
```

Comment:
Passing `locale` into `formatAmountBasedOnLocale` in all places doesn't look very DRY. 
I would `import i18n from '../../i18n';` in `src/utils/formattedNumber.js` and default to `i18n.language` if `locale` is not provided. 

I know it makes `formatAmountBasedOnLocale` less pure, but I think it's worth it.